draft-ietf-core-senml.md

```diff
@@ -718,63 +726,77 @@ number of this specification.
+| Wb       | weber                                             | float | RFC-AAAA  |
+| T        | tesla                                             | float | RFC-AAAA  |
+| H        | henry                                             | float | RFC-AAAA  |
+| Cel      | degrees Celsius                                   | float | RFC-AAAA  |
+| lm       | lumen                                             | float | RFC-AAAA  |
+| lx       | lux                                               | float | RFC-AAAA  |
+| Bq       | becquerel                                         | float | RFC-AAAA  |
+| Gy       | gray                                              | float | RFC-AAAA  |
+| Sv       | sievert                                           | float | RFC-AAAA  |
+| kat      | katal                                             | float | RFC-AAAA  |
+| m2       | square meter (area)                               | float | RFC-AAAA  |
+| m3       | cubic meter (volume)                              | float | RFC-AAAA  |
+| l        | liter (volume)*                                   | float | RFC-AAAA  |
+| m/s      | meter per second (velocity)                       | float | RFC-AAAA  |
+| m/s2     | meter per square second (acceleration)            | float | RFC-AAAA  |
+| m3/s     | cubic meter per second (flow rate)                | float | RFC-AAAA  |
+| l/s      | liter per second (flow rate)*                     | float | RFC-AAAA  |
+| W/m2     | watt per square meter (irradiance)                | float | RFC-AAAA  |
+| cd/m2    | candela per square meter (luminance)              | float | RFC-AAAA  |
+| bit      | bit (information content)                         | float | RFC-AAAA  |
+| B        | Byte (information content, note 3)                | float | RFC-AAAA  |
+| bit/s    | bit per second (data rate)                        | float | RFC-AAAA  |
+| lat      | degrees latitude (note 2)                         | float | RFC-AAAA  |
+| lon      | degrees longitude (note 2)                        | float | RFC-AAAA  |
+| pH       | pH value (acidity; logarithmic quantity)          | float | RFC-AAAA  |
+| dB       | decibel (logarithmic quantity)                    | float | RFC-AAAA  |
+| Bspl     | bel (sound pressure level; logarithmic quantity)* | float | RFC-AAAA  |
+| count    | 1 (counter value)*                                | float | RFC-AAAA  |
+| %        | Percentage (Value of a switch, note 1)*           | float | RFC-AAAA  |
+| %RH      | Percentage (Relative Humidity)*                   | float | RFC-AAAA  |
+| %EL      | Percentage (remaining battery energy level)*      | float | RFC-AAAA  |
+| EL       | seconds (remaining battery energy level)*         | float | RFC-AAAA  |
+| 1/s      | 1 per second (event rate)                         | float | RFC-AAAA  |
+| 1/min    | 1 per minute (event rate)*                        | float | RFC-AAAA  |
```

Comment:
> On Jul 7, 2016, at 3:25 PM, cabo <notifications@github.com> wrote:
> 
> In draft-ietf-core-senml.md <https://github.com/core-wg/senml-spec/pull/6#discussion_r69998945>:
> 
> > +| cd/m2    | candela per square meter (luminance)              | float | RFC-AAAA  |
> > +| bit      | bit (information content)                         | float | RFC-AAAA  |
> > +| B        | Byte (information content, note 3)                | float | RFC-AAAA  |
> > +| bit/s    | bit per second (data rate)                        | float | RFC-AAAA  |
> > +| lat      | degrees latitude (note 2)                         | float | RFC-AAAA  |
> > +| lon      | degrees longitude (note 2)                        | float | RFC-AAAA  |
> > +| pH       | pH value (acidity; logarithmic quantity)          | float | RFC-AAAA  |
> > +| dB       | decibel (logarithmic quantity)                    | float | RFC-AAAA  |
> > +| Bspl     | bel (sound pressure level; logarithmic quantity)* | float | RFC-AAAA  |
> > +| count    | 1 (counter value)*                                | float | RFC-AAAA  |
> > +| %        | Percentage (Value of a switch, note 1)*           | float | RFC-AAAA  |
> > +| %RH      | Percentage (Relative Humidity)*                   | float | RFC-AAAA  |
> > +| %EL      | Percentage (remaining battery energy level)*      | float | RFC-AAAA  |
> > +| EL       | seconds (remaining battery energy level)*         | float | RFC-AAAA  |
> > +| 1/s      | 1 per second (event rate)                         | float | RFC-AAAA  |
> > +| 1/min    | 1 per minute (event rate)*                        | float | RFC-AAAA  |
> I was just afraid that people will be using beats/min for their rotating shaft sensors if we don't define a base 1/min.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/core-wg/senml-spec/pull/6/files/11ef3ac9a8214dcb49a133bea97f5445a6d2c849#r69998945>, or mute the thread <https://github.com/notifications/unsubscribe/AANQcNGUPD4RfKI0Qhpa_HhZLDQkCNNrks5qTXztgaJpZM4JHTeM>.
> 
I think  I would tend to be ok with adding  “rpm” as well as all that to solve the problem.